recipes/conda-build/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "1.21.5" %}
+
+package:
+  name: conda-build
+  version: {{ version }}
+
+source:
+  fn: conda-build-{{ version }}.tar.gz
+  url: https://github.com/conda/conda-build/archive/{{ version }}.tar.gz
+  sha256: af90f4d086377bfa9e9d2be590c1d14c79df58937fb12102b67bf0a5e03032c8
+
+build:
+  number: 0
+  script: python setup.py install
+  entry_points:
+    - conda-env = conda_env.cli.main:main
```

Comment:
I'm confused about this.

I have redone the entry points in the python api branch: https://github.com/conda/conda-build/pull/953/files#diff-2eeaed663bd0d25b7e608891384b7298R40

For these, I have found it necessary to use pip to run the installation.  No matter what arguments I pass to ```python setup.py install```, I always get egg-based entry points.  Pip creates "sane" ones.  I had to rewrite entry points because getting these things for a development install was annoyingly difficult with conda's way of doing it.